pkg/database/databasescriptview.go

```diff
@@ -0,0 +1,195 @@
+package database
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+	proto "github.com/anz-bank/sysl/pkg/sysl"
+
+	"sort"
+	"strings"
+)
+
+type TableDetails struct {
+	table    *proto.Type
+	tableOld *proto.Type
+	action   string
+	name     string
+}
+
+type ScriptView struct {
+	stringBuilder *strings.Builder
+	title         string
+}
+
+type ScriptOutput struct {
+	filename string
+	content  string
+}
+
+func MakeScriptOutput(filename, content string) *ScriptOutput {
+	return &ScriptOutput{
+		filename: filename,
+		content:  content,
+	}
+}
+
+func MakeTableDetails(table, tableOld *proto.Type,
+	action, name string,
+) *TableDetails {
+	return &TableDetails{
+		table:    table,
+		tableOld: tableOld,
+		action:   action,
+		name:     name,
+	}
+}
+
+func MakeDatabaseScriptView(title string,
+) *ScriptView {
+	var stringBuilder strings.Builder
+	return &ScriptView{
+		stringBuilder: &stringBuilder,
+		title:         title,
+	}
+}
+func MakeEmptyDatabaseScriptView() *ScriptView {
+	return &ScriptView{}
+}
+
+func (v *ScriptView) GenerateDatabaseScriptCreate(tableMap map[string]*proto.Type,
+	dbType, appName string) string {
+	v.stringBuilder.WriteString(fmt.Sprintf("/*TITLE : %s*/\n", v.title))
+	v.stringBuilder.WriteString(fmt.Sprint(databaseScriptHeader))
+	appHeader := "\n\n/*-----------------------Relation Model : " +
+		appName + "-----------------------------------------------*/\n"
+	v.stringBuilder.WriteString(fmt.Sprint(appHeader))
```

Comment:
appHeader is already a string, no need to Sprint.

Reply:
done